<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_DataBaseClass</class>
 <widget class="QWidget" name="Test_DataBaseClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据库性能测试</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="pDBTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pMongoTab">
      <attribute name="title">
       <string>Mongo数据库</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pMongoDocBtn">
           <property name="text">
            <string>录入文档</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pMongoUploadPicBtn">
           <property name="text">
            <string>录入图片</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pMongoQueryDocBtn">
           <property name="text">
            <string>查询数据</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="pMongoEdt"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pMySQLTab">
      <attribute name="title">
       <string>MySQL数据库</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pInsertMySQLBtn">
           <property name="text">
            <string>录入数据</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="pMySQLEdt"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Test_DataBase.qrc"/>
 </resources>
 <connections/>
</ui>
